### ISSUE TITLE
Website favicon only shows if hostname is first element

### ISSUE THREAD

**Issue by adripo** [bitwarden/mobile#1120]
## Describe the Bug

Website Icon (favicon) only shows if website url (hostname) is in the first url element.
If the first url is an app link (eg. androidapp:// ) and the website url as second element, the favicon will not show.

## Steps To Reproduce

1. Go to 'My Vault'
2. Click on 'Add Item'
3. Select type 'Login'
4. Insert random name, username, password
5. In URIs section add first element as 'androidapp://example.com'
6. In URIs section add New URI second element with website hotname 'http://github.com'
7. Save

## Expected Result

Credentials icon from website favicon.

## Actual Result

Default item icon without customization.

## Environment

- Device: Xiaomi
- Operating system: Android 9

**Comment by archit9169** [comment]
I dont think this behavior is limited to only mobile. Browser extension also have the same behavior.

**Comment by bugQ** [comment]
> Possible fix: Add a check for default fav icon URI

or just iterate until finding a favicon that actually exists
